nltk/classify/api.py

```diff
@@ -38,12 +43,13 @@ class ClassifierI(object):
     Subclasses may define:
       - either ``prob_classify()`` or ``prob_classify_many()`` (or both)
     """
+    @abstractmethod
     def labels(self):
         """
         :return: the list of category labels used by this classifier.
         :rtype: list of (immutable)
         """
-        raise NotImplementedError()
+        pass
```

Comment:
Personally I would not put the ``pass`` if there is a docstring. It looks redundant, in the sense that there is a code that does nothing.